igvc_gazebo/nodes/sim_color_detector/main.cpp

```diff
@@ -0,0 +1,95 @@
+#include <ros/ros.h>
```

Comment:
I would add a comment right here at the top of the file describing what this node does and what elements in the robot's field of view are cast to white in the output image (i.e. lines -> white, obstacles -> white, etc...)